addons/note/data/note_data.xml

```diff
@@ -1,36 +1,34 @@
-<?xml version="1.0"?>
-<openerp>
-  <data>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
 
-    <record model="note.stage" id="note_stage_00">
+    <record id="note_stage_00" model="note.stage">
```

Comment:
Let's have a convention when renaming data: note_stage_data_01. Demo will be: note_stage_demo_01.